engine/flink/components/base/src/main/scala/pl/touk/nussknacker/engine/flink/util/transformer/FlinkBaseComponentProvider.scala

```diff
@@ -42,7 +43,11 @@ object FlinkBaseComponentProvider {
       ComponentDefinition("for-each", ForEachTransformer).withRelativeDocs("BasicNodes#foreach"),
       ComponentDefinition("union", UnionTransformer).withRelativeDocs("BasicNodes#union"),
       ComponentDefinition("dead-end", SinkFactory.noParam(EmptySink)).withRelativeDocs("DataSourcesAndSinks#deadend"),
-      ComponentDefinition("periodic", PeriodicSourceFactory).withRelativeDocs("DataSourcesAndSinks#periodic")
+      ComponentDefinition("periodic", PeriodicSourceFactory).withRelativeDocs("DataSourcesAndSinks#periodic"),
+      ComponentDefinition(
+        name = "decision-table-flink",
```

Comment:
Please don't add a suffix flink/lite. We want to make it transparent for the user. Also after some refactors I'd create one base provider for both of them and get rid of this duplication